src/ontology/mondo-edit.obo

```diff
@@ -20511,9 +20511,9 @@ id: MONDO:0001385
 name: cortical blindness
 def: "Visual impairment due to visual cortex dysfunction." [NCIT:P378]
 synonym: "cerebral visual impairment" EXACT [https://orcid.org/0000-0002-3458-4839]
-synonym: "CVI" EXACT ABBREVIATION [https://orcid.org/0000-0002-3458-4839]
-synonym: "cortical/cerebral visual impairment" EXACT [https://orcid.org/0000-0002-3458-4839]
 synonym: "cortical visual impairment" EXACT [https://orcid.org/0000-0002-3458-4839]
+synonym: "cortical/cerebral visual impairment" EXACT [https://orcid.org/0000-0002-3458-4839]
```

Comment:
@twhetzel we are seeing this because we didn't normalize after these synonyms were added via ontobot. Protege is just reordering the stanzas in the file.

After new annotations are added via ontobot/kgcl, we should either:
1) open Protege and save as and save the file _or_
2) run:
```
sh run.sh make NORM
mv NORM mondo-edit.obo
```

Reply:
in other words, this can be ignored

Reply:
there are so many nuances in Mondo!